docker_templates/entrypoint.sh.j2

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env sh
```

Comment:
With this being a proper template, this could restore the correct `bash` shebang instead of relying on the `ENTRYPOINT` value, right? (which would make it easier for users to combine with their own scripts without having to know that it needs to be `bash entrypoint.sh` instead of just `./entrypoint.sh`)

Reply:
good thinking, I've pushed some further edits - LMKWYT

Reply:
Nice! :grinning: